ui/js/services/perf/compare.js

```diff
@@ -213,41 +213,60 @@ treeherder.factory('PhCompare', [
                 return errors;
             },
 
-            getResultsMap: function(projectName, seriesList, resultSetIds) {
+            getResultsMap: function(projectName, seriesList, resultSetIds_time) {
```

Comment:
Instead of something like `resultSetIds_time` which can have two values, I'd use an object called params and then have it take multiple possible values (e.g. `resultSetIds` or `interval`). I think that would be a bit easier to understand.